<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CCalculationSequenceEditorClass</class>
  <widget class="QDialog" name="CCalculationSequenceEditorClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>245</width>
        <height>544</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Calculation sequence</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="CQtTree" name="treeWidget">
          <property name="animated">
            <bool>true</bool>
          </property>
          <property name="headerHidden">
            <bool>false</bool>
          </property>
          <property name="columnCount">
            <number>2</number>
          </property>
          <attribute name="headerMinimumSectionSize">
            <number>20</number>
          </attribute>
          <attribute name="headerStretchLastSection">
            <bool>false</bool>
          </attribute>
          <column>
            <property name="text">
              <string />
            </property>
          </column>
          <column>
            <property name="text">
              <string />
            </property>
          </column>
          <property name="toolTip">
            <string>Current simulation sequence</string>
          </property>
          
          <property name="whatsThis">
            <string>Current simulation sequence</string>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1,1,1">
          <item>
            <widget class="QPushButton" name="buttonCalculate">
              <property name="text">
                <string>Calculate sequence</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
              <property name="toolTip">
                <string>Automatically recalculate simulation sequence</string>
              </property>
              
              <property name="whatsThis">
                <string>Automatically recalculate simulation sequence</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QToolButton" name="buttonAdd">
              <property name="minimumSize">
                <size>
                  <width>25</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>+</string>
              </property>
              <property name="toolTip">
                <string>Add partition, model or tear stream to the sequence</string>
              </property>
              
              <property name="whatsThis">
                <string>Add partition, model or tear stream to the sequence</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QToolButton" name="buttonRemove">
              <property name="minimumSize">
                <size>
                  <width>25</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>-</string>
              </property>
              <property name="toolTip">
                <string>Remove the selected partition, model or tear stream from the sequence</string>
              </property>
              
              <property name="whatsThis">
                <string>Remove the selected partition, model or tear stream from the sequence</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QToolButton" name="buttonUp">
              <property name="minimumSize">
                <size>
                  <width>25</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>▲</string>
              </property>
              <property name="toolTip">
                <string>Move the selected partition, model or tear stream upwards in the sequence</string>
              </property>
              
              <property name="whatsThis">
                <string>Move the selected partition, model or tear stream upwards in the sequence</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QToolButton" name="buttonDown">
              <property name="minimumSize">
                <size>
                  <width>25</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>▼</string>
              </property>
              <property name="toolTip">
                <string>Move the selected partition, model or tear stream downwards in the sequence</string>
              </property>
              
              <property name="whatsThis">
                <string>Move the selected partition, model or tear stream downwards in the sequence</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>OK</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>CQtTree</class>
      <extends>QTreeWidget</extends>
      <header location="global">QtTree.h</header>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>treeWidget</tabstop>
    <tabstop>buttonCalculate</tabstop>
    <tabstop>buttonAdd</tabstop>
    <tabstop>buttonRemove</tabstop>
    <tabstop>buttonUp</tabstop>
    <tabstop>buttonDown</tabstop>
    <tabstop>buttonOk</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>CCalculationSequenceEditorClass</receiver>
      <slot>close()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>234</x>
          <y>533</y>
        </hint>
        <hint type="destinationlabel">
          <x>244</x>
          <y>469</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>